sql/dj_setup_database.in

```diff
@@ -254,6 +267,9 @@ while getopts ':u:p:qrs' OPT ; do
 		s)
 			USE_SOCKET=1
 			;;
+		d)
```

Comment:
Should we do this without `-d`, e.g. `dj_setup_database dump post-dress`?

Reply:
The reason I didn't do that was consistency with the other flags. I'm fine with changing to that format.

Reply:
I think it makes more sense to specify a file as a command line argument instead of with a flag. All the other options currently specify global behaviour.

Reply:
I've now changed this to an argument instead of a flag.